projects/packages/status/src/class-request.php

```diff
@@ -17,9 +17,11 @@ class Request {
 	 * Determine whether the current request is for accessing the frontend.
 	 * Also update Vary headers to indicate that the response may vary by Accept header.
 	 *
+	 * @param bool $send_vary_headers Whether to send Vary headers.
```

Comment:
Let's add a `@since $$next-version$$ Added $send_vary_header argument.`

Reply:
Oh, nice catch!

Updated in 0015ff9.